flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -75,12 +63,14 @@
  */
 public class ClientTest extends TestLogger {
 
+	@ClassRule
+	public static final MiniClusterResource MINI_CLUSTER_RESOURCE =
+		new MiniClusterResource(new MiniClusterResourceConfiguration.Builder().build());
+
```

Comment:
Cannot find a call to `MINI_CLUSTER_RESOURCE.before()`. Is it intended?

Reply:
`@ClassRule` does that behind the scenes.